src/ontology/imports/ro_import.owl

```diff
@@ -239,6 +239,12 @@ WHERE {
     
 
 
+    <!-- http://purl.obolibrary.org/obo/RO_0002579 -->
```

Comment:
Why is this file still neeeded?

Reply:
It's needed in the pipeline, to create OWLSRC.

Line 110 on uberon.Makefile: 
```
$(OWLSRC): tmp/uberon-merged.owl $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-gocheck $(REPORTDIR)/$(SRC)-iconv $(SCRIPTSDIR)/expand-dbxref-literals.pl
	(...)
		merge -i imports/ro_import.owl -o $@
```

Reply:
Does this still make sense? Is it even kept up to date by the OWLSRC goal?